pfb-analysis/scripts/setup_database.sh

```diff
@@ -6,11 +6,18 @@ NB_POSTGRESQL_DB=pfb
 NB_POSTGRESQL_USER=gis
 NB_POSTGRESQL_PASSWORD=gis
 
+# Set configuration parameters
+su postgres bash -c psql <<EOF
+ALTER SYSTEM SET work_mem TO '4096MB';
```

Comment:
Setting this to always be 4gb doesn't currently work in dev, as the VM only has 4gb of mem assigned to it. I'm inclined to resolve this by defaulting this setting to 2gb and then allowing for it to be bumped via ENV, rather than further increasing the resources allocated to the dev VM (since additional resources would only be necessary when running large analyses, which we likely won't often do in dev).